app/webdav/server/lib/webdavClientAdapter.ts

```diff
@@ -40,6 +40,7 @@ export class WebdavClientAdapter {
 		try {
 			return await this._client.createDirectory(path);
 		} catch (error) {
+			console.error(error);
```

Comment:
if this is important to keep, can you please use `Logger` instead? we should try not use `console` since we cannot control its output.

Reply:
Nope, this were debug logs. Let me modify